packages/x-date-pickers-pro/src/internals/hooks/useDesktopRangePicker/useDesktopRangePicker.tsx

```diff
@@ -188,7 +189,7 @@ export const useDesktopRangePicker = <
       ...slotProps?.tabs,
       rangePosition,
       onRangePositionChange,
-    },
+    } as ExportedBaseTabsProps,
```

Comment:
After [moving the `className`](https://github.com/mui/mui-x/pull/11814/files#diff-0e12297934cda9420e14b60e09250fac612a44e0235e612ffbf0b06a3ab225b6R17) to this type, it started complaining.
While it was an empty type - all was fine.
I've gone with casting for now just as below in order to avoid a bigger typing refactor.